html/infrastructure/urls/resolving-urls/query-encoding/location.sub.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<meta charset={{GET[encoding]}}> <!-- ends up as <meta charset> by default which is windows-1252 -->
+<meta name=variant content="?encoding=x-cp1251">
+<meta name=variant content="?encoding=utf8">
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id=log></div>
+<script>
+function expected(encoding) {
+  return "?" + {
+    "UTF-8": "%C3%BF",
+    "windows-1251": "&%23255;",
```

Comment:
Either change this to align with https://github.com/whatwg/url/pull/386 before landing or rebase #10915 and change it there.